rxtools/source/lib/console.c

```diff
@@ -90,9 +90,9 @@ void ConsoleShow(){
 			   (y >= ConsoleY && y <= ConsoleY + BorderWidth) || 
 			   (y >= ConsoleH + ConsoleY - 1 && y <= ConsoleH + ConsoleY - 1 + BorderWidth) ||
 			   (y >= ConsoleY + titley - BorderWidth && y <= ConsoleY + titley)){
-				DrawPixel(x, y, BorderColor, tmpscreen);
+				DrawPixel(x, y, BorderColor, (int)tmpscreen);
 			}else{
-				DrawPixel(x, y, BackgroundColor, tmpscreen);
+				DrawPixel(x, y, BackgroundColor, (int)tmpscreen);
```

Comment:
DrawPixel should use a pointer type instead of integer, just because it is a screenbuffer!